Sources/StreamChatUI/MessageActionsPopup/MessageActionsTransitionController.swift

```diff
@@ -148,12 +148,10 @@ open class ChatMessageActionsTransitionController: NSObject, UIViewControllerTra
             usingSpringWithDamping: 0.7,
             initialSpringVelocity: 4,
             options: [.curveEaseInOut],
-            animations: {
+            animations: { [weak self] in
```

Comment:
there's no memory leaks in animation closure, it is safe to remove this 👍

Reply:
Hmm, do we know this for sure? I always added this but I couldn't be 100% sure 🤔